cli/cfncluster/easyconfig.py

```diff
@@ -167,6 +167,10 @@ def configure(args):
             if value is not None and key is not '__name__':
                 config.set(section['__name__'], key, value)
 
+    # ensure that the directory for the config file exists (because
+    # ~/.cfncluster is likely not to exist on first usage)
+    os.makedirs(os.path.dirname(config_file))
```

Comment:
I think os.mkdirs() raises an error when the directory exists already. We might get into that case if a user runs configure after initial bootstrap on the same node.

Reply:
Doh!  Good catch.